### PR TITLE
even more readable boss

### PR DESCRIPTION
**Pull request by NaejDoree**

### CODE REVIEW

File: config/level1/Boss.py3 [hunk 1]
```diff
@@ -157,19 +155,19 @@ while True:
...
+    if(turnType == 0):
         # We will alternate between vertical and horizontal push to shuffle the grid
         # and change the line or the column we want to push at each push
         if(toggle):
-            print("PUSH "+str(column) + " RIGHT")
+            print("PUSH " + str(column) + " RIGHT")
             column = (column + 1) % boardHeight
         else:
-            print("PUSH "+str(row) + " DOWN")
+            print("PUSH " + str(row) + " DOWN")
```
Comment (MaximeCheramy) [inline config/level1/Boss.py3]: You could also do either:

    print("PUSH", row, "DOWN")

or:

    print(f"PUSH {row} DOWN")

File: config/level1/Boss.py3 [hunk 2]
```diff
@@ -157,19 +155,19 @@ while True:
...
-    for i in items:
-        if(i[0] == quests[0]):
-            goalPos = Vector(i[1], i[2])
+    for item in items:
+        if(item.name == quests[0]):
+            goalPos = item.position
 
     # If it's a push turn
-    if(turn_type == 0):
+    if(turnType == 0):
```
Comment (MaximeCheramy) [inline config/level1/Boss.py3]: no need of parentheses

File: config/level1/Boss.py3 [hunk 3]
```diff
@@ -100,14 +101,14 @@ class Tile:
...
-    playerinfos = []
+    myHeroPos = Vector(0,0)
     quests = []
     items = []
 
     board = {}  # The board is a dictionnary linking the position of a tile to the tile object
 
     # Here we get the type of turn : 0 for push and 1 for move
-    turn_type = int(input())
+    turnType = int(input())
```
Comment (MaximeCheramy) [inline config/level1/Boss.py3]: In Python, we use lower case with underscores for variables and functions:

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.
Variable names follow the same convention as function names.

However, it's not a real issue and it's better to have one style that a mix between snake case and camel case :+1:.

File: config/level1/Boss.py3 [hunk 4]
```diff
@@ -117,34 +118,31 @@ while True:
...
-        if(item_player_id == 0):
-            items.append([item_name, item_x, item_y])
+    numItems = int(input())
+    for i in range(numItems):
+        itemName, itemX, itemY, itemPlayerId = input().split()
+        itemPlayerId = int(itemPlayerId)
+        if(itemPlayerId == 0):
+            items.append(Item(itemName, Vector(int(itemX), int(itemY))))
 
     # Here we get le list of the items we want (and ignore the ones of the other player)
```
Comment (eulerscheZahl) [inline config/level1/Boss.py3]: "le list"?

Comment (NaejDoree) [reply config/level1/Boss.py3]: fixed here : 59fdeffc725f509dd0a3a4440c4662356373f5c1

File: config/level1/Boss.py3 (hunk 1 above)
Comment (NaejDoree) [reply config/level1/Boss.py3]: c310020b6eb301bab436045923b322a0ed8371b5

File: config/level1/Boss.py3 (hunk 2 above)
Comment (NaejDoree) [reply config/level1/Boss.py3]: 1bf3c5738d792dcbe2cf2f5a45f376365d993bde

File: config/level1/Boss.py3 (hunk 3 above)
Comment (CodinGameCommunity) [reply config/level1/Boss.py3]: Resolve this?

Comment (MaximeCG) [reply config/level1/Boss.py3]: Yes.

**Pull request merged**